js/clinicalTimeline.js

```diff
@@ -430,6 +431,7 @@ window.clinicalTimeline = (function(){
       d3.select("overview-rectangle").remove();
       advancedView = true;
       timeline();
+      clinicalTimeline.trimTimeline(maxDays, minDays, getZoomLevel, width, getTickValues, margin, formatTime, daysToTimeObject);
```

Comment:
Maybe good to have some boolean here that allows one to trim or not trim timelines? What if I want the simple view without trimming?